server/src/main/java/org/elasticsearch/index/StandardIOBehaviorHint.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.index;
+
+import org.apache.lucene.store.IOContext;
+
+/**
+ * A hint that no special behavior should be set on open files
+ */
+public enum StandardIOBehaviorHint implements IOContext.FileOpenHint {
+    INSTANCE
```

Comment:
Is it possible to have an ESFileOpenHint or something, where we can gather several open hints ?

Reply:
Could do, but there can only be one hint of each type in an IOContext...

Reply:
We can always add that later if it's useful